src/plugins/intel_gpu/src/runtime/debug_configuration.cpp

```diff
@@ -110,15 +110,16 @@ static void print_help_messages() {
     message_list.emplace_back("OV_GPU_DumpSources", "Dump opencl sources");
     message_list.emplace_back("OV_GPU_DumpLayersPath", "Enable dumping intermediate buffers and set the dest path");
     message_list.emplace_back("OV_GPU_DumpLayers", "Dump intermediate buffers of specified layers only, separated by space");
+    message_list.emplace_back("OV_GPU_DumpLayersResult", "Dump intermediate buffers of result layers only");
```

Comment:
nit: if it is network result, it is not an `intermediate` buffer anymore :)

Reply:
Fixed